### ISSUE TITLE
zend-stdlib hydrators have been moved to zend-hydrator

### ISSUE THREAD

**Issue by TomHAnderson** [doctrine/DoctrineModule#546]
Zend has created a new library and changed the namespaces for hydrators.  This library needs to either move hydrators to an external dependency or update the hydrators to use the new namespace.

See also
https://github.com/phpro/zf-doctrine-hydration-module/issues/20
https://github.com/zfcampus/zf-apigility-doctrine/issues/234

@Ocramius suggested I follow up with @GianArb

**Comment by TomHAnderson** [comment]
Will be fixed with https://github.com/doctrine/DoctrineModule/pull/548

**Comment by svycka** [comment]
@Ocramius, to late for 1.0.0 :-)

**Comment by TomHAnderson** [comment]
@svycka you're right.  The release of 1.0 was a surprise to me seeing this had been tagged for that release and this is a BC break so we'll need a large version increment when it's merged.

**Comment by TomHAnderson** [comment]
:+1: 

This is now becoming a thorn in my paw.  I'm rolling back some libraries dependencies so they're compatible now with ZF2.4 although ZF development is moving far ahead of this roadblock.

What can I do to move forward with zend-hydrator in Doctrine?

**Comment by TomHAnderson** [comment]
Because Apigility has moved into zend-hydrator use this fix must be applied to an older branch which DoctrineModule requires: https://github.com/zfcampus/zf-apigility-admin/commit/c6af90177033fc1d4381a1441041efc0300f4290

**Comment by kersten** [comment]
Any updates on this?

**Comment by TomHAnderson** [comment]
I pushed for it.  DoctrineModule won't budge: https://github.com/doctrine/DoctrineModule/pull/558

**Comment by kersten** [comment]
very sad ... :(

**Comment by jguittard** [comment]
👍

**Comment by TomHAnderson** [comment]
This has been remedied:
https://github.com/API-Skeletons/zf-doctrine-module-zend-hydrator

Implemented in 
https://github.com/phpro/zf-doctrine-hydration-module/releases/tag/v2.0.1

I'm still working it into zf-apigility-doctrine.  Once done I'll roll through all these issues with more details.

cc @veewee

**Comment by TomHAnderson** [comment]
Keep this issue alive until zf-apigility-doctrine supports version 2.0 of  this library.

**Comment by driehle** [comment]
Not relevant anymore, as the hydrators now live at [doctrine/doctrine-laminas-hydator](https://github.com/doctrine/doctrine-laminas-hydator) and, besides, have already been ported to Laminas.
